cpp/src/arrow/compute/kernels/vector_selection_test.cc

```diff
@@ -1897,8 +1961,10 @@ TEST(TestFilter, RandomString) {
 }
 
 TEST(TestFilter, RandomFixedSizeBinary) {
-  FilterRandomTest<>::Test(fixed_size_binary(0));
-  FilterRandomTest<>::Test(fixed_size_binary(16));
+  // FixedSizeBinary filter is special-cased for some widths
+  for (int32_t width : {0, 1, 9, 16, 32, 35}) {
```

Comment:
Do we need to use both of `9` and `35`?
Is there any difference for the `9` and `35` cases?

Reply:
You're right, we can probably cut down on the number of combinations here.